NeuroID/src/debug/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -31,7 +32,7 @@ class NeuroID private constructor(
 ) {
     private var isSDKStarted = false
     private var firstTime = true
-    private var endpoint = ENDPOINT_PRODUCTION
+    private var endpoint = ENDPOINT_DEV
```

Comment:
Will implementors use the debug build?  If so, I think the consensus is to use the production endpoint with the TEST environment.

Reply:
Agreed, Updated to use production end point.

Reply:
using prod endpoint in debug